.gitignore

```diff
@@ -43,3 +43,4 @@ build-zip/
 
 # Ignore fast-boot cache file
 /bin/.fast-boot.json
+.fast-boot.json
```

Comment:
Is the `.fast-boot.json` file created on the "current working directory" of the `balena-dev` process? Perhaps a wildcard syntax like `**/.fast-boot.json` (to replace the existing `/bin/.fast-boot.json`) would be suitable.

Reply:
Cool @pdcastro I will go for that :+1: